.ruby-version

```diff
@@ -0,0 +1 @@
+2.2
```

Comment:
Not sure we want to require Ruby version 2.2.  Will the gem still work with projects that use ruby < 2.2?